cluster-autoscaler/Dockerfile.amd64

```diff
@@ -11,9 +11,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-ARG BASEIMAGE=gcr.io/distroless/static:nonroot-amd64
-FROM $BASEIMAGE
+FROM golang:1.23 AS builder
+
+WORKDIR /workspace
+
+COPY . .
+
+ARG ARCH amd64
+
+RUN GOOS=linux GOARCH=$ARCH go build -o cluster-autoscaler-$ARCH
```

Comment:
you'll need to also pass in `$LDFLAGS_FLAGS` and `$TAGS_FLAGS` to match the Makefile build behavior

Reply:
yep, ptal!